lib/vdsm/alignmentScan.py

```diff
@@ -1,23 +1,7 @@
-#
-# Copyright (c) 2012-2017 Red Hat, Inc.
 # Copyright (c) 2012 Sasha Tomic <tomic80@gmail.com>
-#
-# This program is free software; you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation; either version 2 of the License, or
-# (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program; if not, write to the Free Software
-# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
-#
-# Refer to the README and COPYING files for full details of the license
-#
+# Copyright (c) 2012-2017 Red Hat, Inc.
```

Comment:
Can be removed, SPDX-FileCopyrightText replaced it.

If you want to keep the years of the old entry for some reason (I don't see why) you can add multiple SPDX-FileCopyrightText entries.

Reply:
But if removing require manual change to all files, keep it. The important thing is to have new spdx line without year and never touch the year again.